tests/repo_origin/.git-orig/config

```diff
@@ -0,0 +1,5 @@
+[core]
```

Comment:
how much did you learn about git doing this?

Reply:
i just made a simple dumb repo with the following properties and checked it in
master branch - just a A file
featB branch - A file with B file
feat C branch - A file with C file